Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -32,6 +32,18 @@ public class FocusProvider : BaseDataProvider, IMixedRealityFocusProvider
         private PointerHitResult hitResult3d = new PointerHitResult();
         private PointerHitResult hitResultUi = new PointerHitResult();
 
+        /// <summary>
+        /// Number of IMixedRealityNearPointers that are active (IsInteractionEnabled == true).
+        /// </summary>
+        public int NumNearPointersActive { get; private set; }
+
+        /// <summary>
+        /// The number of pointers that support far interaction (like motion controller rays, hand rays) that 
+        /// are active (IsInteractionEnabled == true), excluding the gaze cursor
+        /// </summary>
+        public int NumFarPointersActive { get; private set; }
```

Comment:
Nit: I understand the motivation here but I'll just say it hurts a bit to see the non-universal concepts of near and far pointers hard-coded into the focus provider. I had hoped the pointer mediator could handle the business of near and far pointers somehow. Ah well.

Reply:
@provencher asked to expose the number of near and far pointers. 

I had also thought about putting the near and far pointer management in the mediator, but wanted to avoid the number of classes that knew about / managed the gaze pointer.

If we put the near / far pointer counts in the mediator, how would a dev access it? MRTK.InputSystem.PointerMediator.NumFarPointers? Then we would need to add PointerMediator to the IMixedRealityInputSystem interface, or we could do same trick I do here and cast it to MixedRealitYinputSystem.